### ISSUE TITLE
Better set shuffle=False for the test_loader

### ISSUE THREAD

**Issue by iCurious-D** [aladdinpersson/Machine-Learning-Collection#188]
As I work through the .py files in Machine-Learning-Collection/tree/master/ML/Pytorch/Basics,
I noticed that most of them,  when creating the test_loader,  the shuffle parameter is set to true.
_train_loader = DataLoader(dataset=train_dataset, batch_size=batch_size, shuffle=True)
test_loader = DataLoader(dataset=test_dataset, batch_size=batch_size, shuffle=True)_

But for the test_loader,  it is generally better not to use shuffle=True. 
Not shuffling the test data makes it easier to track and understand the model's performance. 
For instance, if you are reviewing predictions for specific samples, a fixed order allows you to locate these samples more conveniently.

Thank you for your amazing job, really learnt a lot from this repo.
